packages/create-svelte/shared/+eslint+prettier/package.json

```diff
@@ -3,7 +3,7 @@
 		"eslint-config-prettier": "^8.5.0"
 	},
 	"scripts": {
-		"lint": "prettier --plugin-search-dir . --check . && eslint .",
-		"format": "prettier --plugin-search-dir . --write ."
+		"lint": "prettier --cache --check-search-dir . --check . && eslint .",
```

Comment:
why was this changed to `--check-search-dir`? I don't see that in the docs: https://prettier.io/docs/en/plugins.html

Reply:
That was a search-replace error on my side :see_no_evil: . It's fixed now.